### PR TITLE
add onChecked() function to Timeline command

### PR DESCRIPTION
**Pull request by blackmiaool**

### CODE REVIEW

**Comment by blackmiaool** [comment]
image

File: src/app/commands/cmd_timeline.cpp [hunk 1]
```diff
@@ -28,7 +28,8 @@ class TimelineCommand : public Command {
 protected:
   void onLoadParams(const Params& params) override;
   void onExecute(Context* context) override;
-
+  bool onChecked(Context* ctx);
```
Comment (dacap) [inline src/app/commands/cmd_timeline.cpp]: Would you like to add "override" on this one? (like onLoadParams and onExecute). (PS. thanks for the patch :smile:)

Comment (blackmiaool) [inline src/app/commands/cmd_timeline.cpp]: Sorry for ignoring it.

Comment (dacap) [inline src/app/commands/cmd_timeline.cpp]: No problem :+1: I'm here to review these commits and help you :wink:

Comment (blackmiaool) [inline src/app/commands/cmd_timeline.cpp]: Fixed

**Pull request merged**